client/extensions/woocommerce/store-sidebar/store-ground-control.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+const Gridicon = require( 'gridicons' ),
+	Site = require( 'blocks/site' );
+
+import Button from 'components/button';
+
+const StoreGroundControl = ( { site, translate } ) => {
+	const backLink = '/stats/day/' + site.slug;
+
+	return (
+		<div className="store-sidebar__ground-control">
+			<Button
+				borderless
+				className="store-sidebar__ground-control-back"
+				href={ backLink }
+				aria-label={ translate( 'Go back' ) }
+			>
+				<Gridicon icon="arrow-left" />
+			</Button>
+			<div className="store-sidebar__ground-control-site">
+				<Site
+					compact
+					site={ site }
+					indicator={ false }
+					homeLink={ true }
```

Comment:
Fixed in https://github.com/Automattic/wp-calypso/pull/13837/commits/5213d1766e95f25588cf319dc7a1b8f1f2c48e9a